### ISSUE TITLE
RCS1090 does not catch await on Task variables

### ISSUE THREAD

**Issue by gimmi** [JosefPihrt/Roslynator#565]
**Product and Version Used**: Roslynator.Analyzers 2.1.0

**Steps to Reproduce**:

* Enable `RCS1090` with the following configuration:

```xml
<Rules AnalyzerId="Roslynator.CSharp.Analyzers" RuleNamespace="Roslynator.CSharp.Analyzers">
  <Rule Id="RCS1090" Action="Error" />
</Rules>
```

**Actual Behavior**:
```csharp
await MyCodeAsync(); // This generate compilation error RCS1090

var task = Task.Run(MyCodeAsync);
await task; // This compile
```

**Expected Behavior**:
```csharp
await MyCodeAsync(); // This generate compilation error RCS1090

var task = Task.Run(MyCodeAsync);
await task; // This also generate compilation error RCS1090
```

**Comment by gimmi** [comment]
`Roslynator.Analyzer` is still at version `2.1.0` on NuGet (sell [LINK](https://www.nuget.org/packages/Roslynator.Analyzers/)) and, as I understand, this fix is available from version `2.1.3`.

Any plan to push the updated version to nuget.org ?

**Comment by JosefPihrt** [comment]
I have a plan to release new version of nuget Roslynator.Analyzers in ~ 1 week.

**Comment by JosefPihrt** [comment]
https://www.nuget.org/packages/Roslynator.Analyzers/2.2.0-rc
